tests/test_coremanager.py

```diff
@@ -349,7 +349,6 @@ def test_virtual_non_deterministic_virtual(caplog):
         core_manager=cm,
         work_root=work_root,
     )
-    edalizer.run()
```

Comment:
Is this an intentional change, and if so, is it relevant to the other code or should it be in a separate commit?

Reply:
It is intentional, but not relevant to the other changes. I will leave it out.